test/jdk/java/awt/Mixing/AWT_Mixing/GlassPaneOverlappingTestBase.java

```diff
@@ -123,20 +147,17 @@ public void run() {
             }
             Point lLoc = testedComponent.getLocationOnScreen();
             lLoc.translate(1, testedComponent.getPreferredSize().height + 1);
-
-            /* this is a workaround for certain jtreg(?) focus issue:
-               tests fail starting after failing mixing tests but always pass alone.
-             */
-            Util.waitForIdle(robot);
-            ancestorLoc.translate(isOel7orLater() ? 5 : f.getWidth() / 2 - 15, 2);
-            robot.mouseMove(ancestorLoc.x, ancestorLoc.y);
-            Util.waitForIdle(robot);
-            robot.mousePress(InputEvent.BUTTON1_MASK);
-            robot.delay(50);
-            robot.mouseRelease(InputEvent.BUTTON1_MASK);
-            Util.waitForIdle(robot);
-
-            clickAndBlink(robot, lLoc);
+            try {
+                boolean await = latch.await(1, TimeUnit.SECONDS);
+                if (!await) {
```

Comment:
```suggestion
                if (!latch.await(1, TimeUnit.SECONDS)) {
```
I see no need for the variable.